### PR TITLE
[docs] Added License information in Readme.MD

### PR DESCRIPTION
**Pull request by Soumya44**
Added MIT License information and License link to Readme.MD

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,3 +116,7 @@ Guolin Ke, Qi Meng, Thomas Finley, Taifeng Wang, Wei Chen, Weidong Ma, Qiwei Ye,
 Qi Meng, Guolin Ke, Taifeng Wang, Wei Chen, Qiwei Ye, Zhi-Ming Ma, Tie-Yan Liu. "[A Communication-Efficient Parallel Algorithm for Decision Tree](http://papers.nips.cc/paper/6380-a-communication-efficient-parallel-algorithm-for-decision-tree)". Advances in Neural Information Processing Systems 29 (NIPS 2016), pp. 1279-1287.
 
 Huan Zhang, Si Si and Cho-Jui Hsieh. "[GPU Acceleration for Large-scale Tree Boosting](https://arxiv.org/abs/1706.08359)". SysML Conference, 2018.
+
+## License
+
+This project is licensed under the terms of the MIT license. See [LICENSE](LICENSE) for addtional details.
```
[the remark this answers is not in the document]
Comment (Soumya44) [reply README.md]: > Please specify full path to the file.

@StrikerRUS here the file name is ''LICENSE". So "./LICENSE" or "LICENSE" is the same thing. But I'll refine my code as per standard. Thanks for reviewing my PR.

**Comment by Soumya44** [comment]
@StrikerRUS I have updated the link path.

**Comment by Soumya44** [comment]
@StrikerRUS Travis CI build failed due to some other error. Please check the log file. 

>URL        `https://www.microsoft.com/en-us/download/details.aspx?id=49926'
Name       `MS MPI'
Parent URL file:///home/travis/build/Microsoft/LightGBM/docs/_build/html/Installation-Guide.html, line 382, col 42
Real URL   https://www.microsoft.com/en-us/download/details.aspx?id=49926
Check time 1.323 seconds
Size       75.31KB
Result     Error: 404 Not Found

**Comment by StrikerRUS** [comment]
@Soumya44 Sorry for this error. Of course, it's unrelated to your changes. I've proposed hotfix: #1799.

**Comment by StrikerRUS** [comment]
@Soumya44 Sorry for the mess - I've restarted all CI tests.

**Pull request merged**